zephyr/rz/rz_cfg/fsp_cfg/rzg/r_intc_irq_cfg.h

```diff
@@ -0,0 +1,17 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+#ifndef R_INTC_IRQ_CFG_H_
+#define R_INTC_IRQ_CFG_H_
+#ifdef __cplusplus
+            extern "C" {
+            #endif

```

Comment:
@nhutnguyenkc , Please recheck the code alignment here

Reply:
@duynguyenxa , I have fixed the alignment. Please help me check again.